mautrix/client/api/events.py

```diff
@@ -275,11 +279,13 @@ async def send_state_event(self, room_id: RoomID, event_type: EventType,
             https://matrix.org/docs/spec/client_server/r0.5.0#put-matrix-client-r0-rooms-roomid-state-eventtype-statekey
         """
         content = content.serialize() if isinstance(content, Serializable) else content
+        API_CALLS.labels(method="sendStateEvent").inc()
         resp = await self.api.request(Method.PUT, Path.rooms[room_id].state[event_type][state_key],
```

Comment:
@Half-Shot Do I need to wrap this `request()` in a `try` block to catch network failures?

… or will `resp` just be undefined?

Reply:
I'd wrap in a try catch, but remember to just do `raise` so we allow it to fall through [like so](https://github.com/mautrix/signal/pull/164/files#diff-eea437b12b4747e9a300ff993e0194b85fe48c11b2b318410b72d5cbe63d2873R144-R151)